### PR TITLE
Stats: Added check on the parameters of F Distribution

### PR DESCRIPTION
**Pull request by supreet11agrawal**
F Distribution takes two parameters as input which are both positive and
integers. Added checks for this in `crv_types.py` and relevant tests in
`test_continuous_rv.py` of stats module

#### References to other Issues or PRs

Improves #16183

#### Other comments
Docs have not been changed. If needed can be changed. Please tell me

#### Release Notes

* stats
  * Improved paramter handling by F Distribution

### CODE REVIEW

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -1020,6 +1020,11 @@ class FDistributionDistribution(SingleContinuousDistribution):
 
     set = Interval(0, oo)
 
+    @staticmethod
+    def check(d1, d2):
+        _value_check(d1 > 0 and d1.is_integer, "Degrees of freedom d1 must be positive integer.")
```
Comment (jksuom) [inline sympy/stats/crv_types.py]: The line could be split with the string on the next line.

Comment (supreet11agrawal) [reply sympy/stats/crv_types.py]: Ok, I'll do it

**Comment by supreet11agrawal** [comment]
@jksuom should I include changes in docs as well?

**Comment by jksuom** [comment]
What kind of changes in docs do you mean?

**Comment by supreet11agrawal** [comment]
I am saying changes like removing `(n_1 - 1)` and `(n_2 - 1)` from
https://github.com/sympy/sympy/blob/c5a10f9282773572de333c907d75d282508f8483/sympy/stats/crv_types.py#L1055-L1056
I don't think they have any meaning

**Pull request merged**